### PR TITLE
logoff: add page

### PR DESCRIPTION
**Pull request by owenvoke**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/logoff.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Terminate the current session:
+
+`logoff`
+
+- Terminate a session by its name or id:
+
+`logoff {{session_name|session_id}}`
+
+- Terminate a session on a specific RDP server:
```
Comment (agnivade) [inline pages/windows/logoff.md]: maybe change it to - "a specific server connected through RDP" ?

File: pages/windows/logoff.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`logoff /server:{{servername}}`
+
+- Terminate a session within a server or virtual machine (session id is required):
```
Comment (agnivade) [inline pages/windows/logoff.md]: If you say "server or virtual machine", then should it be /{{server|vm}} ?

Comment (pxgamer) [reply pages/windows/logoff.md]: `/vm` is the actual flag.

Comment (agnivade) [reply pages/windows/logoff.md]: alright, so should the description say server also ? Or rather then the question is what is the /server flag for ? Only remote hosts ?

File: pages/windows/logoff.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`logoff`
+
+- Terminate a session by its name or id:
+
+`logoff {{session_name|session_id}}`
+
+- Terminate a session on a specific server connected through RDP:
+
+`logoff /server:{{servername}}`
```
Comment (sbrl) [inline pages/windows/logoff.md]: How does it know which session to terminate on the remote server?

File: pages/windows/logoff.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- Terminate a session within a server or virtual machine (session id is required):
+
+`logoff /vm {{session_id}}`
```
Comment (sbrl) [inline pages/windows/logoff.md]: So if my machine is a VM, I need this extra flag?

File: pages/windows/logoff.md (hunk 3 above)
Comment (pxgamer) [reply pages/windows/logoff.md]: It needs to be specified, I'll add that in.

File: pages/windows/logoff.md (hunk 4 above)
Comment (pxgamer) [reply pages/windows/logoff.md]: I'm going to remove this, it's very rarely used and probably not necessary in a tldr page. :+1:

**Comment by pxgamer** [comment]
@sbrl, I've committed some changes to resolve these.
https://github.com/tldr-pages/tldr/compare/master...pxgamer:ea2279f108d0a53995f958a1965dd08f62814f40

For some reason it's not running Travis on them, or even showing them in the PR. 🤔

**Comment by agnivade** [comment]
Maybe make a dummy change and push again ?

**Pull request merged**